ansible/playbooks/windows.yml

```diff
@@ -74,7 +74,20 @@
         path: C:\temp
         state: directory
       tags: basic_config
-        
+
+    - name: Test if C:\openjdk directory exists
+      win_stat:
+        path: 'C:\openjdk'
+      register: openjdk_installed
+      tags: basic_config
+
+    - name: Create C:\openjdk directory
+      win_file:
+        path: C:\openjdk
+        state: directory
+      when: (openjdk_installed.stat.exists == false)
+      tags: basic_config
+
```

Comment:
This section of code is redundant ```Test if C:\openjdk directory exists```
as ```win_file``` automatically does the same thing. If the folder is already there it will skip.
lines 78-82 and 88 can be removed

Reply:
removed

Reply:
also added clean up for the *.zip files